2018-06-11 10:11:53,242 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 10:37:01,166 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 10:38:18,051 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 10:45:42,000 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 10:52:16,949 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 10:53:19,021 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 10:54:15,208 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 10:55:00,779 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 10:55:51,612 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:00:46,681 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:27:22,300 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:28:07,678 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:29:49,027 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:30:00,890 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:30:52,495 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:41:52,211 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:42:04,721 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:42:48,714 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:45:30,034 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:45:57,426 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:47:38,462 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:48:11,367 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:53:24,990 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:55:11,036 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:57:00,569 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:58:13,265 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:58:37,751 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 11:59:00,163 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:00:53,699 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:03:59,276 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:04:19,808 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:04:48,227 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:05:01,377 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:06:30,476 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:08:27,652 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:10:13,604 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:11:14,968 ERROR: Exception on /films [GET] [in c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Maksim\RecommendationSystem\app\routes.py", line 268, in films
    pop_matrix[i, j] = 0
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\scipy\sparse\lil.py", line 325, in __setitem__
    self.rows, self.data, i, j, x)
  File "_csparsetools.pyx", line 63, in scipy.sparse._csparsetools.lil_insert
  File "_csparsetools.pyx", line 84, in scipy.sparse._csparsetools.lil_insert
IndexError: row index (37) out of bounds
2018-06-11 12:12:53,081 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:12:55,125 ERROR: Exception on /films [GET] [in c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Maksim\RecommendationSystem\app\routes.py", line 268, in films
    pop_matrix[i, j] = 0
  File "c:\users\maksim\recommendationsystem\venv\lib\site-packages\scipy\sparse\lil.py", line 325, in __setitem__
    self.rows, self.data, i, j, x)
  File "_csparsetools.pyx", line 63, in scipy.sparse._csparsetools.lil_insert
  File "_csparsetools.pyx", line 84, in scipy.sparse._csparsetools.lil_insert
IndexError: row index (37) out of bounds
2018-06-11 12:16:28,456 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:18:51,537 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:20:58,143 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:22:12,141 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:22:31,043 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:22:43,330 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:23:17,858 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:23:40,468 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:25:00,364 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:26:11,951 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:27:57,929 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:28:32,417 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:28:49,181 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:29:11,793 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:35:52,574 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:39:21,273 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:46:37,349 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:48:13,158 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:48:42,395 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:50:04,419 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:51:12,751 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:51:32,931 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
2018-06-11 12:52:44,931 INFO: app startup [in C:\Users\Maksim\RecommendationSystem\app\__init__.py:45]
